playbook.md

```diff
@@ -827,31 +819,92 @@ TODO. This section to cover:
 * The probation period, meeting and expectations
 * ...more things.
 
-### The buddy system
-The buddy system is the main way we help each other to perform well and improve our skills. During your first couple of months at dxw, as you get to know the team, you should choose someone and ask them to be your buddy.
 
-As soon as you have a buddy, you'll meet regularly with them to talk about how you're doing. You can talk about things you'd like to improve, problems you're having or new things you'd like to do. After you've discussed what you want to achieve, you'll decide together how best to go about it. In your regular meetings, you'll talk about what's working well and what's not, and decide if you need to change something.
+## Working arrangements
+### Time
+Our working hours are 10:00-18:00, Monday to Friday. Some people work different hours by arrangement. Anyone is free to do that as long as their hours of work don't make it hard for other people to get things done. For example, many people arrive earlier than 10:00 and leave earlier, which is generally fine.
 
-During each meeting, we add cards to the [buddy trello](https://trello.com/b/TmiYBPZg/buddy-chats), so that the rest of the team know what you're working on and can help. If you want to talk about something private, you can. You don't have to put everything on the board. But the presumption is that we do, unless there's a good reason not to.
```

Comment:
I'd like to have a broader conversation about the buddy system and the purpose of it - lots of feedback from the retreat about learning and development, and it came up again in a retro. So won't suggest changes to this but I think we do need to talk about it.